app/models/bounty.rb

```diff
@@ -5,6 +5,7 @@ class Bounty < ApplicationRecord
   has_rich_text :description
 
   scope :sorted, -> { in_order_of(:status, STATUSES).order(created_at: :desc) }
+  scope :last_fifteen_open_bounties, -> { where(status: "open").last(15) }
```

Comment:
I think this can be a little more flexible.  If we add something that only returns open bounties, we can chain additional queries on to that depending on our needs.

`scope :open, -> { where(status: "open") }`

Would let us do `Bounty.open` for all open bounties, `Bounty.first(5)` get the first 5 open Bounties (these are probably the oldest), or `Bounty.open.last(15)` to get the same results as the `last_fifteen_open_bounties` scope above.

After updating the scope, you'll need to make sure all the places you're using that scope is updated